_data/sponsors.yml

```diff
@@ -23,100 +23,133 @@
 
 
 - tier_name: Company Sponsors
-  size: small 
+  size: small
   description: |-
-    These are the companys are sponsor us.
+    These are the company's are sponsor us.
```

Comment:
check spelling / typos, both here and throughout

long term, perhaps we can get grammar check integrated with github actions

Reply:
This should be `companies`.
I also think this is a redundant description.  It is the sponsors page & the `tier_name` describes this.  It should be evident that these companies that sponsor the team.  Consider removing this description.